bokeh/models/widgets/widget.py

```diff
@@ -7,8 +7,11 @@
 from __future__ import absolute_import
 
 from ...core.properties import abstract
+from ...core.properties import String, List
 from ..component import Component
 
 @abstract
 class Widget(Component):
     """ A base class for all interactive widget types. """
+
+    css_classes = List(String)
```

Comment:
It would be convenient to allow space separated strings as well. Use `.accepts(String, lambda str: [ s.trim() for s in str.split("\s+") ])`.